autoload/slime/config.vim

```diff
@@ -43,6 +43,9 @@ if slime#config#resolve("target") == "neovim"
     "delimiter of menu if configuring that way
     let g:slime_config_defaults["neovim_menu_delimiter"] = ', '
 
+    "if set to to a true value, will disable ability to send to unlisted buffers
+    let g:slime_config_defaults["neovim_ignore_unlisted"] = 0
```

Comment:
can you check if the `neovim` defaults can be moved to `autoload/slime/targets/neovim.vim`?

- closer to where needed
- load only if needed